libraries/adaptive-expressions/src/expressionFunctions.ts

```diff
@@ -3355,7 +3355,22 @@ export class ExpressionFunctions {
                 ReturnType.Object, ExpressionFunctions.validateAtLeastOne),
             new ExpressionEvaluator(ExpressionType.JPath, ExpressionFunctions.applyWithError((args: any[][]): any => this.jPath(args[0], args[1].toString())),
                 ReturnType.Object, (expr: Expression): void => ExpressionFunctions.validateOrder(expr, undefined, ReturnType.Object, ReturnType.String)),
+            new ExpressionEvaluator(ExpressionType.Merge, 
+                ExpressionFunctions.applySequenceWithError(
+                    (args: any[]): any => {
+                        let value: object = undefined;
+                        let error: string = undefined;
+                        if ((typeof(args[0]) === 'object' && !Array.isArray(args[0])) && (typeof(args[1]) === 'object' && !Array.isArray(args[1]))) {
+                            Object.assign(args[0], args[1]);
+                            value = args[0];
+                        } else {
+                            error = `The argumets ${ args[0] } and ${ args[1] } must be JSON object.`;
```

Comment:
>object [](start = 93, length = 6)

The arguments _ and _ must be JSON objects.

Reply:
fixed